cmd/kubeadm/app/util/runtime/runtime.go

```diff
@@ -102,15 +102,13 @@ func (runtime *DockerRuntime) IsRunning() error {
 
 // ListKubeContainers lists running k8s CRI pods
 func (runtime *CRIRuntime) ListKubeContainers() ([]string, error) {
-	out, err := runtime.exec.Command("crictl", "-r", runtime.criSocket, "pods", "-q").CombinedOutput()
+	out, err := runtime.exec.Command("crictl", "-r", runtime.criSocket, "pods", "--namespace=kube-system", "-q").CombinedOutput()
```

Comment:
I don't think it's good idea to filter here at all. In case of Docker, there are possibilities that containers are started by user. In case of CRI usage, we can easily assume that all the containers belong to Kubernetes.

Reply:
OK, thanks. will do.